package.json

```diff
@@ -2,12 +2,13 @@
   "name": "libxmljs",
   "author": "Marco Rogers",
   "contributors": [
-    "Jeff Smick"
+    "Jeff Smick",
+    "Martin von Gagern <Martin.vGagern@gmx.net>"
```

Comment:
Cause this isn't where this project tracks contributors. I could just remove this entry altogether. We don't really have a separate contributors file etc for tracking. A git log on the repo or a git blame can show who is responsible for which file.

Maybe people make contributions and they are all appreciated. Attribution is provided by your commits :)